com.github.sunsided.k8sfwd.yml

```diff
@@ -0,0 +1,40 @@
+---
+app-id: com.github.sunsided.k8sfwd
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: k8sfwd
+finish-args:
+  - "--share=network"
+  # - "--filesystem=xdg-config"
+  # - "--filesystem=home"
+  - "--filesystem=host"
```

Comment:
this is a very broad filesystem permission. Your explanation is not convincing. 

If you need some SDK then it should be in the flatpak.

Reply:
It's less about the SDK than about the fact that the tool searches the current directory hierarchy for configuration files. It's a development tool and is supposed to access files outside its sandbox.

`kubectl` can be installed either as part of the Google Cloud SDK (which isn't flatpak'd as far as I know) or separately, depending on what the developer chooses. Specific versions are required for the Kubernetes version; I arguably do not know whether this would work if it could be bundled, but `kubectl` itself isn't flatpak'd either.

Reply:
I understand the need for restricted permissions by the way, I simply don't know a better way of doing this. If I can at least clearly announce that this tool accesses the local file system, I would feel better about it.

Reply:
at that point installing your tool as `cargo install` would be better.

Command line tool are not the use case for flatpak, so the ergonomics are far from optimum.

Reply:
Yeah, I agree. The `cargo install` part requires the toolchain to be installed, making it harder to build (let alone trustworthy if I distribute my own builds). [neovim](https://github.com/flathub/io.neovim.nvim) made a compelling case (including `--filesystem=host`).